packages/atlas/atlas.config.yml

```diff
@@ -54,31 +54,31 @@ features:
       - tier: 'bronze'
         reqs:
           - Lower effort production.
-          - Growing subscriber count base of channel supporters.
+          - Growing subscriber base of channel supporters.
         rewards:
           - 0 # Reward for signup in USD
           - 0 # Reward for synced video
-          - 0 # Reward for referral
+          - 50 # Reward for referral
```

Comment:
We need it to remain correct amount for specific tier, since referral table will need it. We should just use diamond referral amount for UI purposes